src/modules/interface.test.js

```diff
@@ -1,4 +1,5 @@
 import Interface from './interface.js';
+import Storage from './storage.js';
 
 describe('addTask', () => {
   // eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] `required`
- [ ] Amazing job so far
- [ ] please enable the disabled eslint because it defeats the purpose of installing linters. 🥂

Reply:
Okay. Thanks.